### PR TITLE
feat: rename and re-work variables

### PR DESCRIPTION
**Pull request by chrismaree**
this PR re-works a number of re-name and variable changes proposed in the most recent code reading session.

### CODE REVIEW

File: packages/core/contracts/optimistic-asserter/implementation/OptimisticAsserter.sol [hunk 1]
```diff
@@ -6,7 +6,7 @@ import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 
 import "../interfaces/OptimisticAsserterCallbackRecipientInterface.sol";
 import "../interfaces/OptimisticAsserterInterface.sol";
-import "../interfaces/SovereignSecurityInterface.sol";
```
Comment (Reinis-FRP) [inline packages/core/contracts/optimistic-asserter/implementation/OptimisticAsserter.sol:9]: In the comments we still have references to sovereign security. Should those be part of separate PR when adding NatSpec?

**Review by md0x — APPROVED** [review]
👌

File: packages/core/contracts/optimistic-asserter/implementation/OptimisticAsserter.sol [hunk 2]
```diff
@@ -216,22 +216,23 @@ contract OptimisticAsserter is OptimisticAsserterInterface, Lockable, Ownable {
...
+
             address bondRecipient = resolvedPrice == 1e18 ? assertion.asserter : assertion.disputer;
 
-            // If set to use the DVM as oracle then must burn half the bond amount. Else, if not using the DVM as oracle
-            // then the bond is returned to the correct party (asserter or disputer).
-            // TODO:  think of better variable name than "burn". "oracle fee" might be better.
-            uint256 burn = !assertion.ssSettings.arbitrateViaSs ? (burnedBondPercentage * assertion.bond) / 1e18 : 0;
-            uint256 send = assertion.bond * 2 - burn;
+            // If set to use the DVM as oracle then oracleFee must sent to the oracle. Else, if not using the DVM then
+            // the bond is returned to the correct party (asserter or disputer).
```
Comment (Reinis-FRP) [inline packages/core/contracts/optimistic-asserter/implementation/OptimisticAsserter.sol:226]:
```suggestion
            // If set to use UMA DVM as oracle then oracleFee must be sent to UMA store contract. Else, if not using UMA
            // DVM then the bond is returned to the correct party (asserter or disputer).
```

**Review by Reinis-FRP — APPROVED** [review]
